launch/setup.launch.py

```diff
@@ -22,6 +22,7 @@ def declare_launch_arguments():
 
 def generate_launch_description():
     robot_description = ParameterValue(Command(['xacro ', LaunchConfiguration('robot_description_file')]), value_type=str)
+    robot_description_semantic = ParameterValue(Command(['cat ', LaunchConfiguration('robot_description_semantic_file')]), value_type=str)
```

Comment:
Note: in ROS2 Iron+ there is a `FileContent` substitution that could be used here instead